index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <div class="toolbar">
+      <div class="mylogo">My Logo</div>
+      <div class='hamburger'>
+        <div class='line'></div>
+        <div class='line'></div>
+        <div class='line'></div>
+      </div>
+    </div>
```

Comment:
I suggest using HTML5 semantic tags (e.g., header, nav) over divs to Follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), so 
- [ ] the main container carrying the header section should be a `<header>` tag. kindly fix that.

- [ ] The hamburger button is on the right should have a `<nav>` tag

- [ ] `<div class="mylogo">My Logo</div>` this element is supposed to be the logo in this case, so the project is more semantic let's use an `<a>` stage instead of `<div>`
> if you notice on most websites, the logo is usually a link, so you can go back to the homepage by clicking it. That kind of UX convention makes the navigation on the site more pleasant.